plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/extensions/IrBodyFiller.kt

```diff
@@ -218,6 +220,21 @@ private class DataFrameFileLowering(val context: IrPluginContext) : FileLowering
         return declaration
     }
 
+    private fun IrDeclarationWithName.nestedName() = StringBuilder().let { computeNestedName(this, it); it.toString() }
```

Comment:
Would `buildString { computeNestedName(this@nestedName, this) }` not be a lot shorter? :)